docs/development/contribution.md

```diff
@@ -0,0 +1,114 @@
+# Contributors Guide
+
+## Contributing 
+
+The Gardener on Metal project uses Github to manage reviews of pull requests.
```

Comment:
The project name is not "on Metal" anymore. Can you please grep the whole new documentation in this PR with "on metal". (Case insensitive so you get all matches) I think there are more occurrences.